debian/build.go

```diff
@@ -354,6 +359,28 @@ func getKubeletCNIVersion(v version) (string, error) {
 	return fmt.Sprint("= 0.5.1"), nil
 }
 
+// getCRIToolsVersion assumes v coming in is >= 1.11.0-alpha.0
+func getCRIToolsVersion(v version) (string, error) {
+	sv, err := semver.Make(v.Version)
+	if err != nil {
+		return "", err
+	}
+
+	v1110, err := semver.Make("1.11.0-alpha.0")
+	if err != nil {
+		return "", err
+	}
+	v1111, err := semver.Make("1.11.1-alpha.0")
+	if err != nil {
+		return "", err
+	}
+
+	if sv.GTE(v1110) && sv.LT(v1111) {
```

Comment:
is this right? are we going to backport the cri-tools bump to Kubernetes v1.11 as well? I thought we were only possibly discussing v1.12.1.

Reply:
looking at the rpm spec I think this is maybe just a typo? :)

Reply:
Oh, this should be >= 1.11.0-alpha.0 and < 1.12.0-alpha.0